server/functions.lua

```diff
@@ -196,8 +196,15 @@ function PaycheckInterval()
     SetTimeout(QBCore.Config.Money.PayCheckTimeOut * (60 * 1000), PaycheckInterval)
 end
 
--- Callbacks
+-- Callback Functions --
 
+-- Client Callback
+function QBCore.Functions.TriggerClientCallback(name, source, cb, ...)
+    QBCore.ClientCallbacks[name] = cb
+    TriggerClientEvent('QBCore:Client:TriggerClientCallback', source, name, ...)
+end
+
+-- Server Callback
 function QBCore.Functions.CreateCallback(name, cb)
```

Comment:
I think you should change the previous callback natives which would make it a breaking commit to match the naming schema. TriggerServerCallback/CreateServerCallback. Other than that I cannot wait for this to get merged

Reply:
I thought about that as well. Seems like it seemed like too large of a breaking commit to do so but will see what everyone things.